devel/bloaty/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+name                bloaty
```

Comment:
You can start your Portfile off with `github.setup`.

It will implicitly set `name`, `version`, `homepage`, and `master_sites`.

So the following line:
```
github.setup        google bloaty 1.1 v
```

...means you can then delete the `name`, `version`, `homepage` and `master_sites` lines.

Reply:
Updated as suggested 👍